python/tvm/relay/frontend/pytorch.py

```diff
@@ -2357,6 +2357,36 @@ def _impl(inputs, input_types):
     return _impl
 
 
+def _bincount():
+    def _impl(inputs, input_types):
+        data = inputs[0]
+        weights = inputs[1]
+        maximum = _op.max(data)
+        dim = maximum + _expr.const(1, dtype="int64")
+        if weights:
+            out_dtype = "float32"
```

Comment:
While int64 is fixed in PyTorch without weights, passing in float64 weights returns float64, so we should probably take the weight's dtype.

Reply:
thanks, fixed